test/clojure/data/csv_test.clj

```diff
@@ -1,9 +1,12 @@
 (ns clojure.data.csv-test
   (:use
    [clojure.test :only (deftest is)]
+   [clojure.test.check.clojure-test :refer [defspec]]
+   ;[clojure.test.check.generators :as gen]
```

Comment:
Requiring this namespace, without any other changes in the file, will cause `lein test` to fail with `First arg to bind must be a generator` for all tests. Any ideas @AdamClements?

Reply:
It turns out this was caused by the `:use` rather than `:require`, which I belated spotted when tweaking the tests.